test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
As a sanity check, I grabbed lshw output from a Dell device with one gpu (not one of our regular GPU servers) and added two new tests (there are four GPU tests total). Everything works as expected: 7212e38df521774b6aa584a96529d5efadf4fec8

I'm reasonably confident that there is something off with our GPU servers / our lshw output.